intents/util/enums.py

```diff
@@ -48,3 +48,19 @@ class Granularities(enum.Enum):
     MONTHLY = 2
     DAILY = 3
     HOURLY = 4
+
+class Filters(enum.Enum):
+    """The attributes of this class like Filters.EQUALTO ,
+    .NOTEQUALTO , etc. are enum members
+
+    The Filters are listed in an arbitrary order.
+    And currently =, !=, <, <=, >, >=, in, not in are used
+    """
+    EQUALTO = 1
```

Comment:
Can we use _ between words here for readability eg: EQUAL_TO, NOT_EQUAL_TO

Reply:
Yes, that will be better, changed.